### PR TITLE
[BACK-2220] Add Blood Glucose Meter stats

### PR DESCRIPTION
**Pull request by Roukoswarf**
TODO:

- [x] make summary update properly update a summary with only one type filled, not all users have all types, and summaries are updated per-type in platform.

### CODE REVIEW

**Review by toddkazakov — CHANGES_REQUESTED** [review]
Filter parsing needs to be improved because the number of filters grew signifficantly

**Review by toddkazakov — DISMISSED** [review]

File: api/gen_server.go [hunk 1]
```diff
@@ -31,9 +31,6 @@ type ServerInterface interface {
 	// Get Clinic by Share Code
 	// (GET /v1/clinics/share_code/{shareCode})
 	GetClinicByShareCode(ctx echo.Context, shareCode string) error
-
-	// (DELETE /v1/clinics/{clinicId})
-	DeleteClinic(ctx echo.Context, clinicId ClinicId) error
```
Comment (toddkazakov) [inline api/gen_server.go:36]: This shouldn't be deleted

**Review by toddkazakov — CHANGES_REQUESTED** [review]

File: api/gen_server.go (hunk 1 above)
Comment (Roukoswarf) [reply api/gen_server.go:36]: Good spot, it was on the todo list, i did not notice when DeleteClinic got merged.

**Review by toddkazakov — APPROVED** [review]

**Review by toddkazakov — APPROVED** [review]

**Review by toddkazakov — APPROVED** [review]

**Pull request merged**